osquery/tables/system/darwin/smbios_tables.cpp

```diff
@@ -281,5 +282,62 @@ QueryData genPlatformInfo(QueryContext& context) {
   CFRelease(details);
   return {r};
 }
+
+QueryData genAarch64PlatformInfo(QueryContext& context) {
+  auto device_tree =
+      IORegistryEntryFromPath(kIOMasterPortDefault, "IODeviceTree:/");
+  if (device_tree == 0) {
+    return {};
+  }
+
+  CFMutableDictionaryRef details = nullptr;
+  IORegistryEntryCreateCFProperties(
+      device_tree, &details, kCFAllocatorDefault, kNilOptions);
+  IOObjectRelease(device_tree);
+
+  if (details == nullptr) {
+    return {};
+  }
+  Row r;
+  r["vendor"] = getIOKitProperty(details, "manufacturer");
+
+  auto chosen =
+      IORegistryEntryFromPath(kIOMasterPortDefault, "IODeviceTree:/chosen");
+  if (chosen != 0) {
+    IORegistryEntryCreateCFProperties(
+        chosen, &details, kCFAllocatorDefault, kNilOptions);
+    IOObjectRelease(chosen);
+    r["version"] = getIOKitProperty(details, "system-firmware-version");
```

Comment:
I don't know what correct is, but on my m1 machine this PR return `iBoot-7459.121.3` which seems a little weird? And I notice that's not what system profiler says:
```
$ system_profiler SPHardwareDataType | grep -i "Version"
      System Firmware Version: 7459.121.3
      OS Loader Version: 7459.121.3
```

Reply:
I feel like we should strip `iBoot` but I could be convinced it's not worth it, and to see how it does in the broader world first

Reply:
I think the System Information.app application is trying to report less information to the user to make things simpler to understand, but that is definitely an exception.

I personally prefer the output as-is, which is what any other tool will report (such as ioreg) by just reading and returning the actual data without modifications